### ISSUE TITLE
Various components  of the reports are not keyboard accessible

### ISSUE THREAD

**Issue by csmith-he** [archesproject/Arches_HER#1029]
Steps to Reproduce:
1. Navigate to Resource Editor
2. Tab through page contents and attempt to reach the following components using the keyboard:
a. 'Geometry Metadata button
b. 'Reply'/'delete' icons

Expected Result:
User should be able to access all components using a keyboard that a user reliant on a mouse would be able to access. 

Actual Result:
 'Geometry Metadata button and 'reply'/'delete' icon are not keyboard accessible.
